### PR TITLE
NK-222 bug fix - inaugurator will fail only if target installation di…

### PR DESCRIPTION
**Pull request by guymenahem**
…sk failed in SMART test and not any storage device

### CODE REVIEW

File: inaugurator/storagedevices.py [hunk 1]
```diff
@@ -54,7 +54,15 @@ def readSmartDataFromAllDevices(cls, talkToServer=None, failOnFailedHealthTest=F
                     if failOnFailedHealthTest:
                         if talkToServer is not None:
                             talkToServer.healthTestFailed(device)
-                        raise ex
+
+                        # check if device exist in failOnDevices
+                        if any([(device in cur_dev) for cur_dev in failOnDevices]):
```
Comment (eranco74) [inline inaugurator/storagedevices.py:59]: Why not
```suggestion
                        if device in failOnDevices:
```

Comment (eranco74) [reply inaugurator/storagedevices.py:59]: So please add a comment and consider documenting it in a docstring

Comment (guymenahem) [reply inaugurator/storagedevices.py:60]: added

Comment (guymenahem) [reply inaugurator/storagedevices.py:60]: added

**Comment by bazulay** [comment]
approved

**Pull request merged**